controllers/flowlogspipeline/flp_objects.go

```diff
@@ -177,6 +177,8 @@ func (b *builder) podTemplate(configDigest string) corev1.PodTemplateSpec {
 			}},
 			Containers:         []corev1.Container{container},
 			ServiceAccountName: constants.FLPName,
+			HostNetwork:        hostNetwork,
+			DNSPolicy:          corev1.DNSClusterFirst,
```

Comment:
As already [commented in another thread](https://github.com/netobserv/network-observability-operator/pull/78#discussion_r870131906):
* You don't need to use the host network. I could confirm in a manual deployment that (1) the [FLP Daemonset can be deployed without `hostNetwork: true`](https://github.com/netobserv/netobserv-ebpf-agent/pull/27/files#diff-1935f8d554da7493227ad67d151a3babca67f3a8f049b3d53ca78965592bc885) and (2) [the agent can anyway use the HostIP + FLP Port to only send flows to its own host](https://github.com/netobserv/netobserv-ebpf-agent/pull/27/files#diff-d67b1319dcea073fb864a6f7d9e93ac5d56d34d531779dcf092e39364c72318dR31).
* If anyway you want to use `hostNetwork: true` by any other reason I'm not aware of, you need to set `DNSPolicy: corev1.DNSClusterFirstWithHostNet` to allow FLP to access K8s services by its name, such as `loki` while keeping the host network.